http-verbs-common/src/main/scala/uk/gov/hmrc/http/HeaderCarrier.scala

```diff
@@ -62,7 +63,6 @@ case class HeaderCarrier(
       HeaderNames.akamaiReputation      -> akamaiReputation.map(_.value)
     ).collect { case (k, Some(v)) => (k, v) }
 
-  @deprecated("Provide extra headers to VERB (GET, POST) functions", "13.0.0")
   def withExtraHeaders(headers: (String, String)*): HeaderCarrier =
```

Comment:
Ideally this would still be deprecated, and we would add an `addExtraHeaders` function for internal use (the function adds extra headers, it doesn't replace them as you would expect from the name `withExtraHeaders`).
Given the internal client (`WSRequest`) is in package `play.http` rather than `http.play` we can't make it package private. Rather than correcting the package structure, I have just removed the deprecation.